src/utils/misc.js

```diff
@@ -151,13 +153,73 @@ const entryPoint = dir => {
   return fse.realpathSync(path.resolve(dir, packageJson.main));
 };
 
+const printVersionInfo = context => {
```

Comment:
I like this helper, nice job! I think it's acceptable to print the versions while doing a `push`/`upload`.